docs/sources/style-guide/write-for-developers/index.md

```diff
@@ -0,0 +1,194 @@
+
+The advantages of automating documentation programmatically are well-known, and include increased consistency and a reduction in human-made errors.
+But behind every line of auto-generated content is a human author who is responsible for following the [style guide]({{< relref "../../style-guide/" >}}).
+
+When writing documentation that will be used by an auto-generated program to create publishable content, keep the following things in mind:
+
+An auto-generation tool can parse syntax, but when writing documentation that will be used to create automated content, it is up to you to add actionable insights. 
+- What are the caveats, edge cases, and side effects?
+- What is the bigger picture that is not self-evident in the code? 
+- In short, what is everything that a developer needs to know to use the code?
+
+> **Note:** It might be difficult to integrate auto-generated content with other documentation, such as relevant sections of _Get started_ guides, tutorials, or detailed code examples. You might need to ask a Docs team member to ensure that your content is properly cross-referenced.
+
+### Elements of API references
+
+Properly document the most common elements of an API reference, such as the title, parameters, return values, and so on. The following suggestions will help you to write more complete and consistent documentation:
+
+| Element                                               | Description                                                           |
+| ------------------------------------------------- | ----------------------------------------------------------------------- |
+| Title and description       | The name of the element and a description of it in one or two sentences. Use back ticks (``) for API names, classes, methods, and so forth, so they display in a fixed-width font. |
+| Syntax | The code signature that defines the element. If it is possible to use multiple programming languages, provide the syntax for each. Put the signature in `code font`. |
+| Parameters | If the element has parameters, specify their descriptions, data types, and state whether they are optional or required. Put the parameters in _italic_. |
+| Return values| If the element returns a value, describe the range of possibilities and the data type. |
+| Error codes| Describe errors or exceptions and the conditions under which they occur. If possible, provide a way to resolve the issue. |
+| Comments | Describe any important information that hasn't been previously included in the title, description, syntax, parameters, or return values. For example, you might explain non-obvious context, make a comparison to similar elements, or provide cautionary notes about potential gotchas. |
+
+Additional tips:
+- Remember to write concisely. Don't say "This method adds a user." when "Adds a user." will do. If your linter requires the description to begin with the element name, you may say "The AddUser method adds a user." to avoid an error message.
```

Comment:
i think concision makes this tip stronger. Especially since it's a tip about being concise 😂 

```suggestion
- Remember to write concisely. Don't say "This method adds a user." when "Adds a user." will do.
```

Reply:
@osg-grafana @jdbaldry @sdboyer - In my last commit I fixed the sample input/output for command lines. Sam's tips were essential, but I couldn't get the exact markdown he gave me to work so I used something similar. Also, I couldn't get italic to show up as italic. Please feel free to re-review if you have a chance and make additional suggestions.

Reply:
Hi @josmperez, nothing to add. Please merge this.